### ISSUE TITLE
Data Science with Python : Getting Started with Seaborn

### ISSUE THREAD

**Issue by prathimacode-hub** [girlscript/winter-of-contributing#2407]
Welcome to 'DSWP' Team, good to see you here

This issue will helps readers in giving all the guidance that one needs to learn about Getting Started with Seaborn. Tutorial to Seaborn library and how it's applied using sample code.

To get assigned to this issue, add your serial numbers mentioned in the spreadsheet of "Data Science with Python", the approach one would follow and choice you prefer (Documentation, Audio, Video). You can go with all three or any number of options you're interested to work on.

If you had referred any resources, add them up in "DS Resources". Similarly if you had used datasets, include them in "DS Datasets".

Domain : Data Analysis and Visualization

Mentors Assigned : Shivang Dubey / Ishika Kesarwani

Points to Note :

The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.

"Issue Title" and "PR Title should be the same. Include issue number along with it.

Changes should be made inside the Datascience_With_Python/ directory & Datascience_With_Python branch.

Follow Contributing Guidelines & Code of Conduct before start Contributing.

This issue is only for 'GWOC' contributors of 'DSWP' domain.

All the best. Enjoy your open source journey ahead. 😎

**Comment by prathimacode-hub** [comment]
Bot says you have more than 3 open issues allotted to you. Is it? Do changes addressed for your earlier PR if you haven't and you are getting assigned but haven't provided any details of yours. @developer-metalhead

**Comment by pankaj-Makhijani** [comment]
Hello @prathimacode-hub I want to work on this issue.

**Comment by pankaj-Makhijani** [comment]
I would like to work on this.
Pankaj Makhijani - 432
Batch - 12
Issue No - #2407
Type: Doccumentation
Approach:
1. Introduction to seaborn
2. Why to seaborn over matplotlib
3. Types of plots with example
4. Conclusion

**Comment by developer-metalhead** [comment]
I will write the different steps and approaches of using Seaborn. I will also add the code to the documentation.
I am Kumar Satwik with serial number:160

**Comment by developer-metalhead** [comment]
> Bot says you have more than 3 open issues allotted to you. Is it? Do changes addressed for your earlier PR if you haven't and you are getting assigned but haven't provided any details of yours. @developer-metalhead

I already made suggested changes to that issue. Please do check it out

**Comment by developer-metalhead** [comment]
> Bot says you have more than 3 open issues allotted to you. Is it? Do changes addressed for your earlier PR if you haven't and you are getting assigned but haven't provided any details of yours. @developer-metalhead

I already made suggested changes to that issue. Please do check it out

**Comment by prathimacode-hub** [comment]
@developer-metalhead, Yes true but bot isn't accepting you with a reason that you have more than 3 open issues assigned to yourself. So unfortunately, it has been assigned to other.

**Comment by prathimacode-hub** [comment]
Issue is assigned to @pankaj-Makhijani for documentation

**Comment by anamhayat** [comment]
hello
i would like to work on this
Anam Hayat - 937
Batch - 21
Issue No - #2407
Type: Audio
Approach:
1. Introduction to seaborn
2. Visualizing with seaborn
3. Types of plots
4. conclusion

**Comment by prathimacode-hub** [comment]
Issue is assigned to @anamhayat for audio contribution

**Comment by pankaj-sawnani** [comment]
I would like to work on this.
Pankaj Sawnani - 935
Batch - 21
Issue No - #2407
Type: Video
Approach:

-> Introduction to seaborn
-> Different Types of plots with example
-> Conclusion

**Comment by prathimacode-hub** [comment]
Issue assigned to @pankaj-sawnani for video contribution

**Comment by pankaj-sawnani** [comment]
@prathimacode-hub I have created pull request. Please check it.

**Comment by pankaj-sawnani** [comment]
@prathimacode-hub I have made requested changes and created pull request again.
